test/lib/ufe/testSelection.py

```diff
@@ -81,15 +74,22 @@ def runTestSelection(self, selectCmd):
         paths = [j for i in zipped for j in i]
 
         # Create items for all paths.
-        items = [ufe.Hierarchy.createItem(p) for p in paths]
+        self.items = [ufe.Hierarchy.createItem(p) for p in paths]
+
+        # Clear selection to start off
+        cmds.select(clear=True)
```

Comment:
I moved the block above from the test into the setUp() method so it will be called by each test (including my new one). I store the created scene items.